jetson/percep/artag_detector.hpp

```diff
@@ -20,8 +21,14 @@ class TagDetector {
     cv::Mat rgb;
 
    public:
-    TagDetector();                                                                  //constructor loads dictionary data from file
-    Point2f getAverageTagCoordinateFromCorners(const vector<Point2f> &corners);     //takes detected AR tag and finds center coordinate for use with ZED
-    pair<Tag, Tag> findARTags(Mat &src, Mat &depth_src, Mat &rgb);                  //detects AR tags in a given Mat
-    double getAngle(float xPixel, float wPixel);                                    //finds the angle from center given pixel coordinates
+   //constructor loads dictionary data from file
```

Comment:
loads what dictionary data?

Reply:
It's the alvar dictionary info that defines the bit configurations in the tags we are looking for. We can make that clearer